easybuild/easyconfigs/r/ROOT/ROOT-6.12.06_cling-runtime-sysroot.patch

```diff
@@ -0,0 +1,25 @@
+This patch disables the sysroot feature for cling. 
+
+Reusing existing patch found in Gentoo ebuild for ROOT, see url:
+https://github.com/gentoo/gentoo/blob/master/sci-physics/root/files/root-6.12.06_cling-runtime-sysroot.patch
+
+Patch needed for ROOT to build in EESSI:
+https://github.com/EESSI/software-layer/pull/769
+
+Index: root-6.12.06/interpreter/cling/lib/Utils/Paths.cpp
```

Comment:
I know this is annoying to change as you will also need to update the checksums but...
```suggestion
This patch disables the runtime sysroot feature for cling since we would not be cross-compiling in an EasyBuild context. 

Reusing existing patch found in Gentoo ebuild for ROOT, see url:
https://github.com/gentoo/gentoo/blob/master/sci-physics/root/files/root-6.12.06_cling-runtime-sysroot.patch

Patch needed for at runtime for ROOT in EESSI:
https://github.com/EESSI/software-layer/pull/769

Index: root-6.12.06/interpreter/cling/lib/Utils/Paths.cpp
```

Reply:
That is fine. I was very uncertain as to what to write here anyway.

Reply:
Updated the PR with new commit now